src/components/views/globals/NewVersionBar.js

```diff
@@ -96,7 +96,7 @@ export default React.createClass({
         }
         return (
             <div className="mx_MatrixToolbar">
-                <img className="mx_MatrixToolbar_warning" src="img/warning.svg" width="24" height="23" alt="Warning"/>
+                <img className="mx_MatrixToolbar_warning" src="img/warning.svg" width="24" height="23" />
```

Comment:
Not sure I really see why this is being removed?

Reply:
The text it-self is descriptive enough to be understood as warning. Thus I consider having set something other than an empty string to the alt attribute on this image an over verbosity for screen reader users.